test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/condition/TreeHasItem.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2007-2019 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v 1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributor:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.openshift.reddeer.condition;
+
+import org.eclipse.reddeer.common.condition.AbstractWaitCondition;
+import org.eclipse.reddeer.common.exception.RedDeerException;
+import org.eclipse.reddeer.swt.impl.tree.DefaultTreeItem;
+
+/**
+ * 
+ * @author jkopriva@redhat.com
+ * 
+ */
+public class TreeHasItem extends AbstractWaitCondition {
```

Comment:
This also might go to the RedDeer as well.

Reply:
Issue created in RedDeer: https://github.com/eclipse/reddeer/issues/2034